internal/config/main.go

```diff
@@ -15,6 +15,22 @@ const (
 	defaultCollectorConfigMapEntry = "cwagentconfig.json"
 )
 
+// AnnotationConfig details the resources that have enabled
+// auto-annotation for each instrumentation type.
+type AnnotationConfig struct {
```

Comment:
this is just to create the necessary functions to test the webhook, this is subject to change